client/helpers/modal.js

```diff
@@ -0,0 +1,36 @@
+window.onload = function() {
+  const lightboxImages = window.document.querySelectorAll('.modal')
```

Comment:
you probably don't want to have a lightbox in any case where somebody uses a modal class. Since `modal` might be also used for other purposes, I would at least narrow it to img tags, but better refactor it to something more specific like `lightbox-modal`.

```suggestion
  const lightboxImages = window.document.querySelectorAll('img.modal')
```